MAUI/ProgressBar/Events.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Events in .NET MAUI ProgressBar control | Syncfusion
+description: Learn here all about Events support in Syncfusion .NET MAUI ProgressBar control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Events in .NET MAUI ProgressBar (Progress Bar)
+
+## ProgressChanged
+
+This event is triggered when the progress value is changed. This event contains the following event argument.
+
```

Comment:
* `Progress:` Represents the progress value.

Reply:
done